command.go

```diff
@@ -181,16 +181,49 @@ func (c *Command) parseFlags(args Args) (*flag.FlagSet, error) {
 		return set, set.Parse(append([]string{"--"}, args...))
 	}
 
-	if c.UseShortOptionHandling {
-		args = translateShortOptions(args)
-	}
-
 	if !c.SkipArgReorder {
 		args = reorderArgs(args)
 	}
 
+PARSE:
 	err = set.Parse(args)
 	if err != nil {
+		if c.UseShortOptionHandling {
+			// To enable short-option handling (e.g., "-it" vs "-i -t")
+			// we have to iteratively catch parsing errors.  This way
```

Comment:
Don't use 2 spaces after a period, this is not a typewriter ;)
https://www.cultofpedagogy.com/two-spaces-after-period/

Reply:
I am much older than I look ;-)